transaction/transactor.go

```diff
@@ -222,19 +225,26 @@ func Capture(logger kitlog.Logger) kithttp.RequestFunc {
 			source = host
 		}
 
-		logKVs := []interface{}{logging.MessageKey(), "record",
-			"request", Request{
+		traceID, spanID, ok := candlelight.ExtractTraceInfo(ctx)
+		if !ok {
+			logger.Info("record", zap.Reflect("request", Request{
 				Address: source,
 				Path:    r.URL.Path,
 				Query:   r.URL.RawQuery,
-				Method:  r.Method,
-			},
-			"tid", tid,
-			"satClientID", satClientID,
+				Method:  r.Method}),
+				zap.String("tid", tid), zap.String("satClientID", satClientID),
+			)
+		} else {
+			logger.Info("record", zap.Reflect("request", Request{
+				Address: source,
+				Path:    r.URL.Path,
+				Query:   r.URL.RawQuery,
+				Method:  r.Method}),
+				zap.String("tid", tid), zap.String("satClientID", satClientID),
+				zap.String(candlelight.TraceIdLogKeyName, traceID), zap.String(candlelight.SpanIDLogKeyName, spanID),
+			)
 		}
-
-		logKVs, _ = candlelight.AppendTraceInfo(ctx, logKVs)
-		transactionInfoLogger := kitlog.WithPrefix(transactionInfoLogger, logKVs...)
+		transactionInfoLogger, _ := ctx.Value(ContextKeyTransactionInfoLogger).(*zap.Logger)
 		return context.WithValue(nctx, ContextKeyTransactionInfoLogger, transactionInfoLogger)
```

Comment:
```suggestion
		return context.WithValue(nctx, ContextKeyTransactionInfoLogger, transactionLogger)
```
If possible, I would change every reference of `transactionInfoLogger` to `transactionLogger`, including the context key.